src/commons/body-message-renderer.jsx

```diff
@@ -87,30 +86,32 @@ function HtmlMessageRenderer({ msgId, body, parts }) {
 		iframeRef.current.contentDocument.open();
 		iframeRef.current.contentDocument.write(`<div>${updatedBody}</div>`);
 		iframeRef.current.contentDocument.close();
-		const imgMap = reduce(
-			parts,
-			(r, v) => {
-				if (!_CI_REGEX.test(v.ci)) return r;
-				r[_CI_REGEX.exec(v.ci)[1]] = v;
-				return r;
-			},
-			{}
-		);
-
-		const images = iframeRef.current.contentDocument.body.getElementsByTagName('img');
 
-		forEach(images, (p) => {
-			if (p.hasAttribute('dfsrc')) {
-				p.setAttribute('src', p.getAttribute('dfsrc'));
-			}
-			if (!_CI_SRC_REGEX.test(p.src)) return;
-			const ci = _CI_SRC_REGEX.exec(p.getAttribute('src'))[1];
-			if ({}.hasOwnProperty.call(imgMap, ci)) {
-				const part = imgMap[ci];
-				p.setAttribute('pnsrc', p.getAttribute('src'));
-				p.setAttribute('src', `/service/home/~/?auth=co&id=${msgId}&part=${part.name}`);
-			}
-		});
+		// TODO: Commented because it is giving error if appointment has attachments
```

Comment:
Can't this be fixed instead of disabling it?

Reply:
> Can't this be fixed instead of disabling it?

Not in this ticket, it needs a dedicated time .Also it's not properly handled in mails app.So, creating a dedicated ticket for both the apps will  be better

Reply:
Is the ticket already there?

Reply:
> Is the ticket already there?

I already mentioned it as bug on bug channel. @giuliano176  can you confirm if ticket is created for it or not?